### DISCUSSION TITLE
Is there still a version for catalina available?

### DISCUSSION THREAD

**Discussion by tosbsas** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#670] [General]
Hi, if you have a question about AlDente, please use the Discussions section of Github: https://github.com/davidwernhart/AlDente/discussions

**Comment by MatthiasKerbl** [comment]
Hi @tosbsas,

AlDente Classic 2.2 should still be compatible with Catalina.
https://github.com/davidwernhart/AlDente/releases/tag/2.2

Cheers,
Matthias

**Comment by tosbsas** [comment]
It does. Thanks :-) 

Ruben 




______________________________
Ruben Gutknecht
Lima, Perú  🇵🇪 
Cel: +51-99349-1515
Voice/Fax: 49 (0) 721-151-379204
http://telegram.me/tosbsas
paypal.me/RubenGutknecht
***@***.***
